var/spack/repos/builtin/packages/nasm/package.py

```diff
@@ -11,11 +11,11 @@ class Nasm(AutotoolsPackage):
     portability and modularity. It includes a disassembler as well."""
 
     homepage = "http://www.nasm.us"
-    url      = "http://www.nasm.us/pub/nasm/releasebuilds/2.13.03/nasm-2.13.03.tar.xz"
+    url      = "https://src.fedoraproject.org/repo/pkgs/nasm/nasm-2.13.03.tar.bz2/sha512/d7a6b4cee8dfd603d8d4c976e5287b5cc542fa0b466ff989b743276a6e28114e64289bf02a7819eca63142a5278aa6eed57773007e5f589e15768e6456a8919d/nasm-2.13.03.tar.bz2"
     list_url = "http://www.nasm.us/pub/nasm/releasebuilds"
     list_depth = 1
 
-    version('2.13.03', 'd5ca2ad7121ccbae69dd606b1038532c')
+    version('2.13.03', '0c581d482f39d5111879ca9601938f74')
```

Comment:
why the checksum change?

Reply:
I just checked the md5sum after downloading it from the new URL and used this one. Guess it won't work with the old one?

Reply:
the question is: Why did the checksum change in the first place?

Reply:
Idk, but it worked and built fine. The old URL is back up again so I do not know if we should change this. 
However, it was already the second or third time this happened and the fedora repo might be more stable in this regard.

Reply:
They are different files due to different compression. Nasm one is tar.xz whereas fedora's is tar.bz2